docs/Cloud_integrations/Device_Integration_Center/overview.md

```diff
@@ -6,83 +6,153 @@ brokers etc. There are two kind of events:
 * Lifecycle - these are related to device life cycle events like creation, update, deletion.
 * Telemetry - these are related to telemetry gathered by device like temperature measurement.
 
-![Overview](./images/data-integration-center-overview.webp)
+<figure markdown>
+  ![Overview](./images/data-integration-center-overview.webp){ align=center }
+  <figcaption>High-level overview of the Data Integration Center integration</figcaption>
+</figure>
 
 ## Templating
 
 Templating feature allows for defining how your payload (or headers) looks like. This enables you to integrate with
 almost all available services, gathering your data according to your preferences.
 
-![Wizard Form Body](./images/wizard-form-body.webp){ width=50% }
+![Wizard Form Body](./images/wizard-form-body.webp){ width=49% } ![Wizard Preview](./images/wizard-preview.webp){ width=49% }
 
 You can modify a body template as long as it is proper `JSON`. If you want a different content type you can use `Text`
-that have no validation at all.
-
-```json
-{
-  "version": "$version",
-  "timestamp": $timestamp,
-  "endpointName": "$endpointName",
-  "domainId": "$domainId",
-  "url": "$url",
-  "value": "$value"
-}
-```
-
-There are values starting with `$` e.g. `$endpointName`. These are called variables.
+option that have no validation at all.
+
+=== "JSON"
+
+    ```json
+    {
+      "version": "$version",
+      "timestamp": $timestamp,
+      "endpointName": "$endpointName",
+      "domainId": "$domainId",
+      "url": "$url",
+      "value": "$value"
+    }
+    ```
+
+=== "TEXT"
+
+    ```text
+    version: $version
+    timestamp: $timestamp
+    endpointName: $endpointName
+    domainId: $domainId
+    url: $url
+    value: $value
+    ```
+
+Note that there are values starting with `$` e.g. `$endpointName`. These are called variables.
 
 ## Variables
 
 Variables are predefined and allow for using some device context in a payload. When sending event, variables are replaced 
 with actual value e.g. `$endpointName` with `test-device-47193512895`. There are several variables that can be used, if
-variable does not exist e.g. `$notExisting` then it is sent as normal text. Available variables differs from context where
-they are used.
+variable does not exist e.g. `$notExisting` then it is sent as normal text.
+
+=== "Device Lifecycle Variables"
+
+      * `version` *(type: string)* - The version in which events are sent. Any changes will be reflected in it.
```

Comment:
The rule is that if the text following the hyphen is not a sentence, then the 1st word shouldn't be capitalized. However, some sentences have a verb, therefore it would be good to unify those definitions.

Reply:
I changed it a bit